src/client/datascience/jupyter/kernels/kernelDependencyService.ts

```diff
@@ -87,13 +90,19 @@ export class KernelDependencyService implements IKernelDependencyService {
         _token?: CancellationToken
     ): Promise<boolean> {
         try {
-            const version = await this.installer.getVersion(Product.ipykernel, interpreter);
-            if (version) {
-                const versionSplit = version.split('.');
-                const mainVersionNumber = Number(versionSplit[0]);
-                return mainVersionNumber >= 6;
+            const result = await this.pythonInstaller.isProductVersionCompatible(
+                Product.ipykernel,
+                '>=6.0.0',
```

Comment:
This isn't a valid semver.
We should ideally just pass `6.0.0` and then on the other end use the necessary API to check the minimum version by prefixing wqith `>=` or the like.

Reply:
It is valid and it works.
There's an example here: https://docs.npmjs.com/cli/v6/using-npm/semver#ranges